eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 ## 1.10.0.0
 
+* Add `psDRepDistrG`.
+* Rename `ensPParams` to `ensCurPParams`.
+* Add `ToJSON` instance for `RatifyState`
+* Change `ToJSON` instance for `ConwayGovState`:
+  * Add `"nextRatifyState"` field
```

Comment:
The backtick + double quote looks a little weird, is it on purpose?  (it doesn't matter too much anyway, just curious)

Reply:
Those are json fields, so they are represented as strings. Backticks will result in formatting as code, which json is:
![image](https://github.com/input-output-hk/cardano-ledger/assets/2333894/abe1bcfc-d3aa-4859-be85-9ace201949e5)

Reply:
Makes sense, thanks